web/packages/teleport/src/Notifications/Notifications.tsx

```diff
@@ -208,8 +208,15 @@ export function Notifications({ iconSize = 24 }: { iconSize?: number }) {
       >
         <ButtonIconContainer
           onClick={onIconClick}
+          onKeyUp={e => e.key === 'Enter' && onIconClick()}
```

Comment:
Thoughts on using `onKeyUp` vs `onKeyDown`? For the most part I see that we use `onKeyDown` more often, but I actually don't know which one is better in general and in this context.

Edit: This answer suggests `onKeyUp` is preferable for actions that are non-repeatable and that don't need to be perceived as fast. https://ux.stackexchange.com/a/151057/33322

Reply:
I typically go with onKeyUp as that's what onClick does, it waits for the mouse to be lifted up before firing, which allows dragging+releasing off of a button if accidentally clicked for example